net-fs/wdfs/wdfs-1.4.2-r3.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="WebDAV filesystem with special features for accessing subversion repositories"
+HOMEPAGE="http://noedler.de/projekte/wdfs/"
+SRC_URI="http://noedler.de/projekte/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+
+RDEPEND="
+	>=net-libs/neon-0.24.7
```

Comment:
Are we maybe missing a slot operator here?

Reply:
That would be your task to find out. The existence of a subslot for `net-libs/neon`
```
$ eshowkw neon
Keywords for net-libs/neon:
             |                               |   u      |  
             | a   a     p s     l r   a     |   n      |  
             | m   r h   p p   i o i s l m m | e u s    | r
             | d a m p p c a x a o s 3 p 6 i | a s l    | e
             | 6 r 6 p p 6 r 8 6 n c 9 h 8 p | p e o    | p
             | 4 m 4 a c 4 c 6 4 g v 0 a k s | i d t    | o
-------------+-------------------------------+----------+-------
   0.32.4-r1 | + + + ~ + + + + ~ ~ ~ ~ ~ o ~ | 8 o 0/27 | gentoo
[I]0.33.0    | ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ o ~ | 8 o      | gentoo
```
...should make you look into the linking of wdfs with e.g. `lddtree`. If you see linking aginst `.27` soname that means wdfs should be triggered for rebuild on `net-libs/neon` subslot bumps. `:=` achieves that.